src/components/network/map-equipments.js

```diff
@@ -222,6 +226,7 @@ export default class MapEquipments {
     }
 
     updateLines(lines, fullReload) {
+        this.waitingForUpdate = false;
```

Comment:
Why does waitingForUpdate is set to false in updateLines and not in updateSubstations ?

Reply:
Because lines are needed too and they are asked for later than substations (and are heavier to load), so I considered that enough data has come to start the redraw at that point.

Reply:
Alright, can you add a comment to contextualize please ?